installation_and_upgrade/ibex_install_utils/tests/test_backup_tasks.py

```diff
@@ -0,0 +1,33 @@
+import io
+import os
+from unittest.mock import patch, Mock
+from ibex_install_utils.tasks.backup_tasks import BackupTasks
+from ibex_install_utils.tasks import BaseTasks
+from ibex_install_utils.user_prompt import UserPrompt
+
+EPICS_PATH_BACKUP = os.path.join(BaseTasks._get_backup_dir(),'EPICS', 'VERSION.txt')
```

Comment:
`BaseTasks._get_backup_dir()` raises an exception on machines without a `C:\\Data` directory, which is not guaranteed to be present on a developer machine. As this is done before running the tests, this prevents them from running instead of failing the test. 

It might be worth running this with `os.path.exists()` mocked out so that whether this test runs is not coupled to whether the machine has a specific folder present.